lib/tags/profile-settings.tag

```diff
@@ -398,13 +406,15 @@
                             self.profile.birthday = birthday;
                             self.refs.birthday.classList.remove('graphjs-error');
                             self.refs.birthday.classList.add('graphjs-success');
+                            self.refs.birthday.setAttribute('data-savesuccesstext', i18n.saveSuccessText);
```

Comment:
Please remove this change. self.refs.birthday refers to an input element and input elements do not accept ::before or ::after pseudo-elements.

Reply:
Same as above